terraform/examples/gcp/base-cluster/daemonset.yaml

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
Can we add a reference  (comment with a link) form where this is coming from? This will help us remember what this is in the future